android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -202,36 +203,40 @@ public void onDownloadStart(String url, String userAgent, String contentDisposit
 
         RNCWebViewModule module = getModule(reactContext);
 
-        DownloadManager.Request request = new DownloadManager.Request(Uri.parse(url));
-
-        String fileName = URLUtil.guessFileName(url, contentDisposition, mimetype);
-        String downloadMessage = "Downloading " + fileName;
-
-        //Attempt to add cookie, if it exists
-        URL urlObj = null;
         try {
-          urlObj = new URL(url);
-          String baseUrl = urlObj.getProtocol() + "://" + urlObj.getHost();
-          String cookie = CookieManager.getInstance().getCookie(baseUrl);
-          request.addRequestHeader("Cookie", cookie);
-        } catch (MalformedURLException e) {
-          System.out.println("Error getting cookie for DownloadManager: " + e.toString());
-          e.printStackTrace();
-        }
+          DownloadManager.Request request = new DownloadManager.Request(Uri.parse(url));
```

Comment:
here if we will add new DownloadManager.Request(Uri.parse(url.replace("blob:", "")), then blob crash can be fixed or else if any other solution is there that will be fine

Reply:
By doing this, it will still download the junk file. It'll not prevent downloading the file, but yes it'll prevent crashing